ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/blocks/BeaconBlockInvariants.java

```diff
@@ -18,16 +18,24 @@
 import org.apache.tuweni.bytes.Bytes;
 import tech.pegasys.teku.infrastructure.ssz.schema.SszType;
 import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+import tech.pegasys.teku.spec.datastructures.blocks.versions.deneb.SignedBlockContents;
 
 public class BeaconBlockInvariants {
 
+  private static final int SSZ_OFFSET_SIZE = 4;
```

Comment:
```suggestion
  private static final int BLOCK_CONTENTS_SSZ_OFFSET = 4;
```

Reply:
I changed it to BYTES_PER_LENGTH_OFFSET to align with the official ssz spec.